frontend/src/app/accountCreation/MfaSelect/MfaSelect.tsx

```diff
@@ -10,18 +11,45 @@ import { accountCreationSteps } from "../../../config/constants";
 export const MfaSelect = () => {
   const [mfaOption, setMfaOption] = useState("");
   const [mfaOptionError, setMfaOptionError] = useState("");
+  const [submitted, setSubmitted] = useState(false);
 
   const validateMfaOption = () => {
+    let error = "";
     if (mfaOption === "") {
-      setMfaOptionError("Select an authentication option");
-    } else {
-      setMfaOptionError("");
+      error = "Select an authentication option";
+    }
+    setMfaOptionError(error);
+    return error === "";
+  };
+
+  const handleSubmit = () => {
+    if (validateMfaOption()) {
+      setSubmitted(true);
     }
   };
 
   const lessSecure = "Less secure";
   const secure = "Secure";
 
+  if (submitted) {
+    switch (mfaOption) {
+      case "SMS":
+        return <Redirect to="/mfa-sms" />;
+      case "Okta":
+        return <Redirect to="/mfa-okta-verify" />;
+      case "Google":
+        return <Redirect to="/mfa-google-auth" />;
+      case "FIDO":
+        return <Redirect to="/mfa-security-key" />;
+      case "Phone":
+        return <Redirect to="/mfa-phone" />;
+      case "Email":
```

Comment:
Can you make the `mfaOption` better-typed (as a string union) if we know these are the options?

Reply:
I'm on it!